solution/ui/regulations/eregs-vite/src/components/SearchInput.vue

```diff
@@ -106,17 +106,17 @@ export default {
     },
 
     created() {
-        this.searchInputValue = this.searchQuery;
+        this.searchInputValue = decodeURIComponent(this.searchQuery);
```

Comment:
encoded URL query parameter is decoded on create if it exists so it appears on the page as a human-readable string